sources/packages/web/src/components/common/StudentOverawardDetails.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <v-card class="mb-5">
+    <v-container>
+      <body-header
+        title="Overaward balances"
+        subTitle="A balance of overawards broken down by award type"
+      />
+      <v-row>
+        <v-col>
+          <title-value
+            :propertyValue="
+              formatCurrency(overawardBalance.overawardBalanceValues?.CSLF, '')
+            "
+          >
+            <template #title>
+              CSLF
```

Comment:
On the Vue side, the loan/grants acronyms can be defined in a const to also be used by `FULL_TIME_AWARDS/PART_TIME_AWARDS`. Looking more into the code, I have a stronger feeling that the below comment makes sense (the part of the bullet).
https://github.com/bcgov/SIMS/pull/1748/files#r1116077791